packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md

```diff
@@ -13,13 +13,11 @@ Before you can sign a user in, you need to create an Okta app integration that r
     > **Note:** It is important to choose the appropriate application type for apps that are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients aren't designed to have, hence breaking the sign-in or sign-out flow.
 1. Enter a name for your app integration (or leave the default value).
 1. Enter values for the **Login redirect URI**. This is the callback described in <GuideLink link="../define-callback">Understand the callback route</GuideLink>. Add values for local development (for example, `http://localhost:8080/authorization-code/callback`) and production (for example, `https://app.example.com/authorization-code/callback`).
-
+1. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow by redirecting to this endpoint (for example, `https://example:0000.com/login`). This triggers the client to send an authorize request. This URI is also used when users reset their passwords and need to be redirected back to your app.
```

Comment:
Why reset password will redirect to the client, which client reset password will redirect to?

Reply:
Please see: https://oktainc.atlassian.net/browse/OKTA-381015 
and
https://github.com/okta/okta-aspnet/issues/170

These are the reason I added that sentence on resetting passwords......

Reply:
Ok, so it is user reset the password during signing into the client, then after reset, okta will redirect to the initiate_login_uri of the client.